simmechanics/CMakeLists.txt

```diff
@@ -93,28 +93,96 @@ macro(generate_icub_simmechanics)
     else()
       set(GAZEBO_ASSIGNED_INERTIAS "")
     endif()
+
+    # If we are generating the icub2.5+ model we need to add the xsens IMU, change the mesh folder
+    # and change the rotation axis list that need to be reversed.
+    if(GIVTWO_ICUB_PLUS)
+      set(MESH_FOLDER "filenameformatchangeext: \"package://iCub/meshes/simmechanics/2-5_plus/%s-binary.stl\"")
+      set(ANKLE_PITCH_ROM "-50,20")
+      set(CUSTOM_EPSILON "epsilon: 2e-7")
+      set(XSENS_IMU_FRAME
+"  - frameName: SCSYS_ROOT_LINK_XSENS_IMU
+    frameReferenceLink: root_link
+    exportedFrameName: root_link_imu_frame
+")
```

Comment:
Since we now have 2 IMUs, could we rename the previous "imu_frame" in the head to "head_imu_frame"?

Reply:
Yes, we can do it. I am not sure if then we need to update also some other configuration files (e.g. wbd). If you think that is easy and quick we can change the name of the head imu as you suggest.

Reply:
I think there a small impact in iDynTree, quite easy to implement, I let @traversaro confirm...

Reply:
We would need to either have different wbd configuration files for `iCub***` and `iCub***_plus`, or update the name for all the models. To be honest, I am a bit afraid about this small changes that may have unexpected consequences, so I would open a different PR for changing the `imu_frame` frame.

Reply:
This would also affect all our controllers that should be updated accordingly @gabrielenava

Reply:
Thanks @diegoferigo, as suggested by @traversaro we don't change the imu name in this PR. We can open a new one.